app/routes.php

```diff
@@ -1,15 +1,19 @@
 <?php
 declare(strict_types=1);
 
+use App\Controllers\CoursesAPIController;
 use Slim\App;
+use Slim\Views\PhpRenderer;
 use Slim\Interfaces\RouteCollectorProxyInterface as Group;
 
 return function (App $app) {
     $container = $app->getContainer();
 
```

Comment:
code comment:
`//demo code - two ways of linking urls to functionality, either via anon function or linking to a controller`

Reply:
added